src/main/cpp/blaze.cc

```diff
@@ -557,14 +557,14 @@ static void AddLoggingArgs(const LoggingInfo &logging_info,
 
   // The time in ms a command had to wait on a busy Blaze server process.
   // This is part of startup_time.
-  if (command_wait_duration_ms.IsUnknown()) {
+  if (!command_wait_duration_ms.IsUnknown()) {
```

Comment:
In the spirit of avoiding double negatives (which I suspect might have contributed to the bug) can we call the method `IsKnown()` instead?

Reply:
Thinking some more about it: if the only consequence of `IsUnknown()` is that we don't set a `--command_wait_time` flag, why not set it to 0 in that case? The flags library can't distinguish between 0 and unset anyway.